netmanager/src/views/components/DataDisplay/Devices.js

```diff
@@ -158,20 +158,28 @@ const createDeviceColumns = (history, setDelState) => [
   {
     title: "Deployment status",
     field: "isActive",
-    render: (data) => (
-      <Cell
-        fieldValue={
-          <span
-            style={{
-              color: data.status === "deployed" ? "green" : "red",
-              textTransform: "capitalize",
-            }}
-          >
-            {data.status}
-          </span>
-        }
-      />
-    ),
+    render: (data) => {
+      const deviceStatus = !data.status
```

Comment:
Same feedback applies here. Perhaps I could be missing something. Happy to hear your thoughts. @Codebmk